### PR TITLE
[ZEPPELIN-3583] Add function getNoteName() in InterpreterContext

### PR DESCRIPTION
**Pull request by egorklimov**
### What is this PR for?
Now we could get only current user and note ID.

```
note_id = z.getInterpreterContext().getNoteId()
user = z.getInterpreterContext().getAuthenticationInfo().getUser()
```
It would be nice if we could also get note name, like:
```
note_name = z.getInterpreterContext().getNoteName()
```

### What type of PR is it?
Improvement

### What is the Jira issue?
issue on Jira: https://issues.apache.org/jira/browse/ZEPPELIN-3583

### How should this be tested?
* CI pass

### Screenshots
screenshot-2018-7-5 here is notebook name - zeppelin

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
@egorklimov Have you tried it in zeppelin ? It doesn't seem correct to me. Because you didn't  pass note name from zeppelin server to interpreter process. It would always be null if you call  `z.getInterpreterContext().getNoteName()`

**Comment by egorklimov** [comment]
Looks like fixed
* CI is green: https://travis-ci.org/TinkoffCreditSystems/zeppelin/builds/400447972
* Screenshot added.

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/thrift/RemoteInterpreterContext.java [hunk 1]
```diff
@@ -65,6 +65,7 @@
   private static final org.apache.thrift.protocol.TField GUI_FIELD_DESC = new org.apache.thrift.protocol.TField("gui", org.apache.thrift.protocol.TType.STRING, (short)8);
   private static final org.apache.thrift.protocol.TField NOTE_GUI_FIELD_DESC = new org.apache.thrift.protocol.TField("noteGui", org.apache.thrift.protocol.TType.STRING, (short)9);
   private static final org.apache.thrift.protocol.TField LOCAL_PROPERTIES_FIELD_DESC = new org.apache.thrift.protocol.TField("localProperties", org.apache.thrift.protocol.TType.MAP, (short)10);
+  private static final org.apache.thrift.protocol.TField NOTE_NAME_FIELD_DESC = new org.apache.thrift.protocol.TField("noteName", org.apache.thrift.protocol.TType.STRING, (short)11);
```
Comment (zjffdu) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/interpreter/thrift/RemoteInterpreterContext.java]: You miss the thrift file change.

**Comment by egorklimov** [comment]
Look again please:
* Thrift file updated
* CI is green: https://travis-ci.org/TinkoffCreditSystems/zeppelin/builds/400831400

**Pull request closed without merging**